web/test/helpers.tsx

```diff
@@ -128,6 +128,10 @@ export const randomIntFromInterval = (min: string, max: string) => {
   return Math.floor(Math.random() * (Number(max) - Number(min) + 1) + Number(min));
 };
 
+export const randomElementFromArray = (array: unknown[]) => {
+  return array[Math.floor(Math.random() * array.length)];
+};
+
```

Comment:
I think you could use generic types here to keep type safety. Like `export const randomElementFromArray = <T, >(array: T[]): T => {`